common/components/WalletDecrypt/index.jsx

```diff
@@ -165,7 +171,7 @@ export class WalletDecrypt extends Component {
 
   onUnlock = (payload: any) => {
     this.props.dispatch(
-      WALLETS[this.state.selectedWalletKey].unlock(this.state.value || payload)
+      WALLETS[this.state.selectedWalletKey].unlock(payload || this.state.value)
```

Comment:
Payload is a SyntheticEvent in some cases. We'll either need to check if payload isn't one, or change how onUnlock is called so that it's never sent an event.

Reply:
I believe I have fixed this in one of my recent PRs https://github.com/MyEtherWallet/MyEtherWallet/pull/183/files#diff-c2a1968700ed6452c7c0257d4dde4cd4R168

Reply:
Sweet, in that case we just need to bring `develop` back into this bad boy.